examples/reconnect.py

```diff
@@ -5,16 +5,14 @@
 from contextlib import suppress
 from datetime import UTC, datetime
 
-from websockets.exceptions import ConnectionClosed, ConnectionClosedOK
+from aiohttp import ServerDisconnectedError
 
 from aiowebostv import WebOsClient
 from aiowebostv.exceptions import WebOsTvCommandError
 
 WEBOSTV_EXCEPTIONS = (
-    OSError,
-    ConnectionClosed,
-    ConnectionClosedOK,
-    ConnectionRefusedError,
+    ServerDisconnectedError,
+    ConnectionResetError,
```

Comment:
Is it `aiohttp.ClientConnectionResetError`, which inherits `ConnectionResetError`, that is raised or is `ConnectionResetError` raised directly?

Reply:
I think that this is a `ConnectionResetError` raised directly. I could only reproduce this once, but this is the stack trace:
```
[11:08:45.261] Connected: False, Powered on: False
Exception in callback _ProactorBasePipeTransport._call_connection_lost(None)
handle: <Handle _ProactorBasePipeTransport._call_connection_lost(None)>
Traceback (most recent call last):
  File "C:\Program Files\Python312\Lib\asyncio\events.py", line 88, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Program Files\Python312\Lib\asyncio\proactor_events.py", line 165, in _call_connection_lost
    self._sock.shutdown(socket.SHUT_RDWR)
ConnectionResetError: [WinError 10054] An existing connection was forcibly closed by the remote host
```